internal/hmyapi/apiv2/harmony.go

```diff
@@ -79,5 +81,7 @@ func (s *PublicHarmonyAPI) GetNodeMetadata() NodeMetadata {
 		header.Epoch().Uint64(),
 		blockEpoch,
 		cfg.Role().String(),
+		cfg.DNSZone,
```

Comment:
why DNSZone is configured in this PR?

Reply:
It was already there in the apiv1, so I copied it over to the v2.